### DISCUSSION TITLE
Is possible to hide some views according to the user who is watching it?

### DISCUSSION THREAD

**Discussion by Marius9595** [community/community#12664] [Projects and Issues]
- It could be customizable by that user because only needs some views in his stuff
- Also it could be by policies access, but these should be more I guess

Anyway, Good job, I like the beta project feauture!

**Comment by mattcox** [comment]
Agreed. It would be nice to be able to specify a user role on a project/repo (QA, Project Owner, Dev...etc), and then hide views based on that role.

**Comment by terrywbrady** [comment]
My project has quickly created over a dozen views.  It could also be useful to allow view tabs to be nested under tab groups to help organize them.
